### PR TITLE
feat: M2-3400 Custom Themes Support: Admin Panel part

### PR DESCRIPTION
**Pull request by IhorNesterenko99**

### CODE REVIEW

File: src/modules/Builder/features/AboutApplet/AboutApplet.tsx [hunk 1]
```diff
@@ -10,31 +10,33 @@ import {
...
-  width: 20,
-  height: 20,
-};
+import { getColorThemeOptions } from './AboutApplet.utils';
+import { commonUploaderProps } from './AboutApplet.const';
 
 export const AboutApplet = () => {
   const { t } = useTranslation();
-
+  const { result: themesArr = [] } = themes.useThemesData() || {};
```
Comment (alex-i-e) [inline src/modules/Builder/features/AboutApplet/AboutApplet.tsx:30]: maybe rename it to themeList or themeArray?

File: src/modules/Dashboard/state/Users/Users.thunk.ts [hunk 2]
```diff
@@ -3,17 +3,13 @@ import { AxiosError } from 'axios';
...
 
 export const getAllWorkspaceRespondents = createAsyncThunk(
   'users/getAllWorkspaceRespondents',
   async ({ params }: GetAppletsParams, { rejectWithValue, signal }) => {
     try {
-      return await getWorkspaceRespondentsApi(
-        { params: { ...params, limit: RESPONDENTS_WITHOUT_LIMIT } },
-        signal,
-      );
+      return await getWorkspaceRespondentsApi({ params: { ...params, limit: MAX_LIMIT } }, signal);
```
Comment (alex-i-e) [inline src/modules/Dashboard/state/Users/Users.thunk.ts:12]: please add a TODO to discuss more suitable solution with BA: pagination or infiniti scroll.

File: src/modules/Library/api/api.ts [hunk 3]
```diff
@@ -34,6 +34,6 @@ export const postAppletsToCartApi = (cartItems: PublishedApplet[], signal?: Abor
 
 export const getAppletsFromCartApi = (signal?: AbortSignal) =>
   authApiClient.get('/library/cart', {
-    params: { limit: CART_ITEMS_WITHOUT_LIMIT },
+    params: { limit: MAX_LIMIT },
```
Comment (alex-i-e) [inline src/modules/Library/api/api.ts:37]: please add a TODO to discuss more suitable solution with BA: pagination or infiniti scroll.

File: src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.tsx [hunk 4]
```diff
@@ -163,7 +163,7 @@ export const AddToBuilderPopup = ({
       const { selectedWorkspace: ownerId } = getValues();
       if (!ownerId) return;
       getWorkspaceApplets({
-        params: { ownerId, limit: APPLETS_WITHOUT_LIMIT, flatList: true },
+        params: { ownerId, limit: MAX_LIMIT, flatList: true },
```
Comment (alex-i-e) [inline src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.tsx:166]: please add a TODO to discuss more suitable solution with BA: pagination or infiniti scroll.

**Review by alex-i-e — APPROVED** [review]

**Review by nmialik — APPROVED** [review]

File: src/modules/Builder/features/AboutApplet/AboutApplet.styles.ts [hunk 5]
```diff
@@ -1,7 +1,8 @@
-import { styled } from '@mui/system';
+import { styled, Box } from '@mui/system';
```
Comment (muhurov) [inline src/modules/Builder/features/AboutApplet/AboutApplet.styles.ts:1]: please change to @mui/material

**Review by muhurov — APPROVED** [review]

**Pull request merged**